lang/Español.json

```diff
@@ -140,7 +140,7 @@
         "tray": "Ejecutar en la bandeja del sistema: ",
         "tray_notifications": "Mostrar notificaciones: ",
         "priority_only": "Minar solo juegos preferidos: ",
-        "prioritze_end": "Priorizar campañas por fecha de finalización: ",
+        "priority_by_ending_soonest": "Priorizar campañas por fecha de finalización: ",
```

Comment:
I'm not quite sure what changing the variable name will do to [GitLocalize](https://gitlocalize.com/repo/9644) as further translations have been added since this was forked, that now include the old var name

edit: I find your settings description more fitting, tho I don't like the idea of it not matching translations as well, but that's fine. As for the var name, I feel like, besides probably breaking already added new translations, it mostly just adds clutter and makes it harder to read.

Reply:
@Windows200000 well `prioritze` is spelled wrong and wasn't being used at all when I opened this PR.
What you say is "clutter" I say is more readable and telling what the variable is used for.

Reply:
> @Windows200000 well `prioritze` is spelled wrong and wasn't being used at all when I opened this PR. What you say is "clutter" I say is more readable and telling what the variable is used for.

Prioritise can be spelled both ways, the z is more American while the s is more British. Also, it was there, in the English and other translations. I added it earlier, when I was working on dark mode, to have more translations ready when the feature is implemented.

As for readability, priority is only ever used when referring to drops in this project. What you propose would be clearer for new contributors, tho there are much worse offenders for that, such as `"tray": "Autostart into tray: "`.

But longer variable names also take me out of reading the code. If I have to comprehend the variable name first, then I sometimes have to read a whole part again to properly understand what is being done. It's kind of like adding a whole sentence in the middle of a "flowing code text".

If you look around the project, and most other open source code for that matter, you will see around 2 words for variables and maybe up to 4 for functions, as they do their own thing and are often an entire line and outside of context. But especially variables need to be understood in context and thusly should be short enough to feel like one thing and to not forget the entire expression the var was a part of. At least that's why I think that's the case. Look at the following 2 code blocks:

Long
```
        if (
            game not in self.games_to_mine
            and game.name not in excluded_games
            and (not only_watch_prioritized or game.name in prioritzed_games)
            and campaign.can_earn_within_next_hour()
        ):
```
Short
```
        if (
            game not in self.wanted_games
            and game.name not in exclude
            and (not priority_only or game.name in priority)
            and campaign.earnable_within_hour()
        ):
```